Source/Initialization/GetVelocity.H

```diff
@@ -0,0 +1,90 @@
+/* Copyright 2021 Hannah Klion
+ *
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#ifndef GET_VELOCITY_H_
+#define GET_VELOCITY_H_
+
+#include "VelocityProperties.H"
+
+/**
+ * \brief Get velocity at a point on the grid
```

Comment:
Simpler with title + body:
```suggestion
/** Get velocity at a point on the grid
```